rasr/config.py

```diff
@@ -259,7 +259,7 @@ def build_config_from_mapping(crp, mapping, include_log_config=True, parallelize
         config._update(crp.log_config)
         post_config._update(crp.log_post_config)
 
-    for mkey in ["corpus", "lexicon", "acoustic_model", "language_model", "recognizer"]:
+    for mkey in ["corpus", "lexicon", "acoustic_model", "language_model", "recognizer", "label_scorer"]:
```

Comment:
If you add this key here, please also initialize it with None in https://github.com/rwth-i6/i6_core/blob/main/rasr/crp.py#L13